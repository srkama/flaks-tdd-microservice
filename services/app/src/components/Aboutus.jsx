import React from 'react'

const Aboutus = () => {
    return (
        <div>
            <h3>About us page</h3>
            <p>
                About us
             </p>
        </div>
    )
}

export default Aboutus
