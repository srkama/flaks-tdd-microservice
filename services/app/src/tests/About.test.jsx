import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer';
import Aboutus from '../components/Aboutus';

it('renders without crashing', () => {
    const wrapper = shallow(<Aboutus />)
    const element = wrapper.find('p')
    expect(element.length).toBe(1);
    expect(element.text()).toBe('About us');
});

test('About us snapshots renders properly', () => {
    const tree = renderer.create(<Aboutus />).toJSON();
    expect(tree).toMatchSnapshot();
})