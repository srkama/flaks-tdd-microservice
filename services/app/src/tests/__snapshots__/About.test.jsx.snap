// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About us snapshots renders properly 1`] = `
<div>
  <h3>
    About us page
  </h3>
  <p>
    About us
  </p>
</div>
`;
